README.md

```diff
@@ -9,7 +9,25 @@ You can see details of [the project roadmap here](https://napari.org/roadmaps/in
 
 ## contributing
 
-Contributions are encouraged! Please read our [contributing guide](https://napari.org/dev/developers/contributing/documentation/index.html) to get started. Given that we're in an early stage, you may want to reach out on our [Github Issues](https://github.com/napari/docs/issues) before jumping in.
+Contributions are encouraged! Given that we're in an early stage, you may want to reach out on our [Github Issues](https://github.com/napari/docs/issues) before jumping in.
```

Comment:
I'm not sure I understand the comment about being in an early stage here - does this mean the contributor is just getting started, or that napari is in early stages? If it's the former maybe we want to say something like "Make sure you check our [issue tracker](https://github.com/napari/docs/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) before jumping in."

Reply:
I guess I'm not sure, though I'd understood it as the latter. I left this sentence as it had been, but I added an additional sentence and reordered the sentences in this paragraph. I was not trying to change its meaning, sorry if the shuffling of sentences has made it more ambiguous 😅 happy to reword if preferred.

Reply:
Yeah this sentence is a bit wierd! 
I think this may have been copy-pasta from the readme of the napari repo which used to be the home of the docs too, so then it made more sense?
Anyhow, I vote making it explicitly addressed to new contributors who happen upon the README vs. the actual docs on napari.org
How about: `If you're new or unsure about anything check out issues tracker or reach out on zulip`
https://napari.zulipchat.com

Reply:
OK, I reword [here](https://github.com/napari/docs/pull/372/commits/f704b39314d9fd8a4aefbdc723c5f519731a5015)